deployment/ansible/roles/raster-foundry.docker/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-docker_compose_version: "1.7.1"
```

Comment:
We can probably move this file entirely and just leave this variable defined in `all` like it is already.